src/Overlay.tsx

```diff
@@ -128,7 +155,15 @@ const Overlay = React.forwardRef<HTMLDivElement, OverlayProps>(
           height={height}
           {...rest}
           ref={combinedRef}
-          visibility={visibility}
+          style={
```

Comment:
Cool use of the [CSS variables + `styled-components` combo](https://www.joshwcomeau.com/css/styled-components/) we recently discussed! ❤️